starter-code/main.js

```diff
@@ -0,0 +1,165 @@
+                            }
+                        })
+                        break;
+                    case "4":
+                        db.collection('companies').find({ "founded_year": { $eq: 2004 } }, { name: 1, _id: 0 }).toArray((error, result) => {
+                            if (error) {
+                                console.log(error);
+                                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                            } else {
+                                console.log(result);
+                                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                            }
+                        })
+                        break;
+                    case "5":
+                        db.collection('companies').find({ $and: [{ 'founded_year': { $eq: 2004 } }, { 'founded_month': { $in: [4, 5, 6] } }] }, { name: 1, _id: 0 }).toArray((error, result) => {
+                            if (error) {
+                                console.log(error);
+                                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                            } else {
+                                console.log(result);
+                                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                            }
+                        })
+                    case "6":
+
+                        db.collection('companies').count({ "offices.city": { $in: ["Barcelona"] } }).then(function (error, result) {
+                            if (error) {
+                                console.log(error);
+                                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                            } else {
+                                console.log(result);
+                                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                            }
+                        })
+                        break;
+                    case "7":
+
+                        db.collection('companies').find({}, { name: 1, _id: 0 }).sort({ number_of_employees: -1 }).limit(10).toArray((error, result) => {
+                            if (error) {
+                                console.log(error);
+                                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                            } else {
+                                console.log(result);
+                                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                            }
+                        })
+                        break;
+                    case "8":
+
+                        db.collection('companies').count({ "name: 1": { $in: ["Barcelona"] } }).then(function (error, result) {
+                            if (error) {
```

Comment:
Creo que te confundiste en esta query, pues solicitaba _"Find the company with the name "Facebook""_
```
...db.collection("companies")
              .find({ name: "Facebook" })
              .toArray((error, result) => { ...
```